pkg/events/usermode.go

```diff
@@ -63,10 +65,10 @@ func fetchInitNamespaces() map[string]uint32 {
 	return initNamespacesMap
 }
 
-func (t *Tracee) CreateExistingContainersEvents() []trace.Event {
+func ExistingContainersEvents(containers *containers.Containers) []trace.Event {
 	var events []trace.Event
-	def := EventsDefinitions[ExistingContainerEventID]
-	for _, info := range t.containers.GetContainers() {
+	def := Definitions.Get(ExistingContainer)
+	for _, info := range containers.GetContainers() {
```

Comment:
Does this really work?
The cgroups field of this instance will be empty, as it was not initialized

Reply:
Shouldn't be different, I just moved the location of the function. As long as do the same initialization logic as before I think it should be fine, since we're passing the same instance of `containers`. And from a double check I did get my running containers.